app/presenters/error_presenter.rb

```diff
@@ -7,11 +7,19 @@ def initialize(error_messages)
 
   def formatted_error_messages
     error_messages.map do |attribute, messages|
-      [attribute, "#{attribute.to_s.humanize} #{messages.first}"]
+      [attribute, formatted_message(messages.first, attribute)]
     end
   end
 
   private
 
+  def formatted_message(message, attribute)
+    if message.match(/\A[A-Z].+\Z/)
```

Comment:
If the message is something custom like 'Please state the reasons why this application is, or is not lawful', return it as is is. If the message is something more Railsey like 'can't be blank', return eg 'Email can't be blank'.